pkg/chunk/aws/storage_client.go

```diff
@@ -713,6 +761,68 @@ func (b dynamoDBWriteBatch) Add(tableName, hashValue string, rangeValue []byte,
 	})
 }
 
+// FIXME: temporary hack passing in storageClient so we can get at the schemaCfg
```

Comment:
FIXME

Also keen not add more dependancies from here to the `chunk` pkg, this dependancy is something I want to remove in the future.

Reply:
Yeah, looks like only used in `PutOne`, so the logic could go there?